adapter/src/debug_session.rs

```diff
@@ -436,7 +436,10 @@ impl DebugSession {
                                 RequestArguments::_symbols(args) =>
                                     self.handle_symbols(args)
                                         .map(|r| ResponseBody::_symbols(r)),
-                                _=> Err("Not implemented.".into())
+                                _=> {
+                                    info!("Received an unknown command: {}", command);
```

Comment:
In your case, it was not an unknown command, it was a failure to parse launch parameters, because they didn't match the schema.   Separating these two cases is a bit tricky with serde, or at least I haven't spent time yet to figure out how to do it.

Reply:
Oh, nevermind, I thought you were the same person who had reported #427

Reply:
It may not be the same problem. My lanuch.json:

```json
{
    // Use IntelliSense to learn about possible attributes.
    // Hover to view descriptions of existing attributes.
    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
    "version": "0.2.0",
    "configurations": [
        {
            "type": "lldb",
            "request": "launch",
            "name": "Debug executable 'hello_world'",
            "cargo": {
                "args": [
                    "build",
                    "--bin=hello_world",
                    "--package=hello_world"
                ],
                "filter": {
                    "name": "hello_world",
                    "kind": "bin"
                }
            },
            "args": [],
            "cwd": "${workspaceFolder}"
        }
  ]
}
```

It looks very simple

Reply:
What do you think its better info is?

Reply:
em...you are right, one of my environment variable values is of type number, thx